packages/yum/test.sh

```diff
@@ -153,13 +160,76 @@ function mroonga_is_registered() {
   sudo systemctl stop ${service_name}
 }
 
+function mroonga_is_registered_for_mysql_community_minimal() {
+  sudo mkdir -p /var/lib/mysql /var/run/mysqld; \
+  sudo chown mysql:mysql /var/lib/mysql /var/run/mysqld; \
+  sudo chmod 1777 /var/lib/mysql /var/run/mysqld
+
+  auto_generated_password=$(mysqld --initialize |& awk 'END{print $NF}')
+  mysql="mysql -u root -p${auto_generated_password}"
+  mysqld &> /dev/null &
+
+  while ! mysqladmin ping -hlocalhost --silent; do sleep 1; done
+
+  sudo ${mysql} --connect-expired-password -e "ALTER USER user() IDENTIFIED BY '$auto_generated_password'"
+
+  sudo ${mysql} < /usr/share/mroonga/install.sql
+  sudo ${mysql} -e "SHOW ENGINES" | grep Mroonga
+}
+
 repositories_dir=/vagrant/packages/${package}/yum/repositories
 sudo ${DNF} install -y \
   ${repositories_dir}/${os}/${major_version}/*/Packages/*.rpm
 
-mroonga_is_registered
+case ${package} in
+  mysql-community-minimal-*)
+    mroonga_is_registered_for_mysql_community_minimal
+    ;;
+  *)
+    mroonga_is_registered
+    ;;
+esac
 
 # Run test
+
+function run_upgrade_test() {
+  sudo ${DNF} erase -y \
+    ${package} \
+    "${mysql_package_prefix}-*"
+  sudo rm -rf /var/lib/mysql
+
+  # Disable upgrade test for first time packages.
+  case ${os}-${major_version} in
+    oracle-linux-*) # TODO: Remove this after 13.01 release.
+      exit
+      ;;
+  esac
+  case ${package} in
+    mariadb-10.11-*) # TODO: Remove this after 13.01 release.
+      exit
+      ;;
+    mysql-community-minimal-*) # TODO: Remove this after 13.02 release.
+      exit
+      ;;
+  esac
+
+  sudo ${DNF} install -y ${old_package}
+  sudo ${DNF} install -y \
+    ${repositories_dir}/${os}/${major_version}/*/Packages/*.rpm
+
+  mroonga_is_registered
+}
+
+case ${package} in
+  mysql-community-minimal-*)
+  # mysql-community-minimal only execute a upgrade test.
+  # Because we can not install mysql-community-test package on the enviromnment that is installed mysql-community-minimal.
+  # Because the mysql-community-minimal package and the mysql-community-test package are conflict.
+    run_upgrade_test
```

Comment:
We don't need to run upgrade test because this package is only used in Docker.
Docker image doesn't use package upgrade.